src/game_character.h

```diff
@@ -581,6 +581,27 @@ class Game_Character {
 	/** @return the direction we would need to face away from hero. */
 	int GetDirectionAwayHero();
 
+	/**
+	 * @return the direction 90 degrees to the left of dir
+	 *
+	 * @param dir input direction
+	 * */
+	static int GetDirection90DegreeLeft(int dir);
+
+	/**
+	 * @return the direction 90 degrees to the right of dir
+	 *
+	 * @param dir input direction
+	 * */
+	static int GetDirection90DegreeRight(int dir);
+
+	/**
+	 * @return the direction 180 degrees to the of dir
+	 *
+	 * @param dir input direction
```

Comment:
no idea if doxygen will mess this up because the normal order is ``@param @return``

Also ``* */`` instead of ``*/``

Reply:
Doxygen doesn't care about the order of tags. I fixed the `* */`